packages/nimble-components/src/wafer-map/specs/README.md

```diff
@@ -89,12 +89,16 @@ _The key elements of the component's public API surface:_
         -   x: number
         -   y: number
         -   value: string
-    -   `quadrant` - represents the orientation of the dies on the wafer map (the layout of the values on the dies). It can be represented by a const(as suggested [here](https://github.com/ni/nimble/blob/12a84ea7ad9103ab848aa2cd9f724e8853751a10/packages/nimble-components/docs/coding-conventions.md#use-const-objects-instead-of-typescript-enums)) with the following values:
+    -   `quadrant` - represents the direction of the two axes, X and Y, which are used for displaying the die grid on the wafer map. The four possible combinations are represented with the position of the quadrant respective to the specific axes directions . It can be represented by a const(as suggested [here](https://github.com/ni/nimble/blob/12a84ea7ad9103ab848aa2cd9f724e8853751a10/packages/nimble-components/docs/coding-conventions.md#use-const-objects-instead-of-typescript-enums)) with the following values:
```

Comment:
I realize I may be late to the game for this particular API, but `quadrant` as a term, doesn't map intuitively to what this feature does. It seems to me all this is really doing is axis inversion, which I think could be represented much more understandably as two separate boolean attributes: `invertX` and `invertY` (or something even more specific, but the `invert` term is clear in its purpose).

Reply:
For me thinking in invert-x or invert-y terms isn't mentally clearer. I think I'd have to toggle the properties until it looks right lol.

I think what would be clearer for me is defining it in terms of the origin location on a "screen". From that perspective I think the names top-left, top-right, bottom-left, bottom-right make sense. The term quadrant doesn't really though because we aren't like focusing on one quadrant of the coordinate system, we are re-mapping the coordinate system.

My proposal would be the name `dies-origin` (or `dies-grid-origin` to match the outcome of the other discussion) and then keep the enum values top-left, top-right, bottom-left, bottom-right, thoughts @atmgrifter00 ?

side note: It's kinda like dealing with game engine coordinates lol

![image](https://github.com/ni/nimble/assets/1588923/1f09a0dc-b65e-4425-9ee8-93309f5e108d)

Reply:
I agree that the naming can create confusion and tbh it took me a while as well to get comfortable with the concept.

I think we may have a misalignment in our goals because we are not actual domain experts in the semiconductor area, so a lot of the jargon may sound strange to us. At the same time this component is targeted towards other developers who may or may not have the domain knowledge, but they may be tasked to implement features using our component.